client/app/pages/alert/alert.html

```diff
@@ -51,7 +51,7 @@
             <div class="form-group">
               <label class="control-label col-md-2">Rearm seconds</label>
               <div class="col-md-4">
-                <input type="number" class="form-control" ng-model="$ctrl.alert.rearm" ng-disabled="!$ctrl.canEdit"/>
+                <input type="number" class="form-control" ng-model="$ctrl.alert.options.rearm" ng-disabled="!$ctrl.canEdit"/>
```

Comment:
`rearm` is a "top level" property of the alert object and not part of the additional options.

The reason it wasn't persisted on a new alert was this: https://github.com/getredash/redash/pull/1848/files#diff-da69e038c153e31118e65fec91fb5548R57.